### ISSUE TITLE
Getting error listing rake tasks with 1.0.0

### ISSUE THREAD

**Issue by ares** [ofedoren/apipie-dsl#2]
reproducer:
```
rake -T apipie                                                     
```

traceback:
```
/home/vagrant/foreman/lib/foreman.rb:8: warning: already initialized constant Foreman::UUID_REGEXP
/home/vagrant/foreman/lib/foreman.rb:8: warning: previous definition of UUID_REGEXP was here
2020-01-31T15:10:28 [I|app|] Rails cache backend: File
rake aborted!
LoadError: cannot load such file -- /home/vagrant/foreman/.vendor/ruby/2.5.0/gems/apipie-dsl-1.0.0/app/helpers/apipie_dsl_helper
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `block in require_with_bootsnap_lfi'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require_with_bootsnap_lfi'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/polyglot-0.3.5/lib/polyglot.rb:65:in `require'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:287:in `block in require'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:253:in `load_dependency'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:287:in `require'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:48:in `require_relative'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/apipie-dsl-1.0.0/lib/apipie_dsl/tasks_utils.rb:4:in `<top (required)>'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `block in require_with_bootsnap_lfi'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require_with_bootsnap_lfi'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/polyglot-0.3.5/lib/polyglot.rb:65:in `require'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:287:in `block in require'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:253:in `load_dependency'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:287:in `require'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:48:in `require_relative'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/apipie-dsl-1.0.0/lib/apipie_dsl/tasks/apipie_dsl/cache.rake:3:in `<top (required)>'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:54:in `load'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:54:in `load'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:281:in `block in load'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:253:in `load_dependency'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:281:in `load'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/apipie-dsl-1.0.0/lib/apipie_dsl/railtie.rb:12:in `block (2 levels) in <class:Railtie>'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/apipie-dsl-1.0.0/lib/apipie_dsl/railtie.rb:12:in `each'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/apipie-dsl-1.0.0/lib/apipie_dsl/railtie.rb:12:in `block in <class:Railtie>'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/railties-5.2.1/lib/rails/railtie.rb:246:in `instance_exec'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/railties-5.2.1/lib/rails/railtie.rb:246:in `block in run_tasks_blocks'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/railties-5.2.1/lib/rails/railtie.rb:255:in `each'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/railties-5.2.1/lib/rails/railtie.rb:255:in `each_registered_block'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/railties-5.2.1/lib/rails/railtie.rb:246:in `run_tasks_blocks'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/railties-5.2.1/lib/rails/application.rb:514:in `block in run_tasks_blocks'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/railties-5.2.1/lib/rails/engine/railties.rb:15:in `each'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/railties-5.2.1/lib/rails/engine/railties.rb:15:in `each'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/railties-5.2.1/lib/rails/application.rb:514:in `run_tasks_blocks'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/railties-5.2.1/lib/rails/engine.rb:459:in `load_tasks'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/railties-5.2.1/lib/rails/railtie.rb:190:in `public_send'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/railties-5.2.1/lib/rails/railtie.rb:190:in `method_missing'
/home/vagrant/foreman/Rakefile:8:in `<top (required)>'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/rake-13.0.1/exe/rake:27:in `<top (required)>'
/opt/rh/rh-ruby25/root/usr/share/gems/gems/bundler-1.16.1/lib/bundler/cli/exec.rb:75:in `load'
/opt/rh/rh-ruby25/root/usr/share/gems/gems/bundler-1.16.1/lib/bundler/cli/exec.rb:75:in `kernel_load'
/opt/rh/rh-ruby25/root/usr/share/gems/gems/bundler-1.16.1/lib/bundler/cli/exec.rb:28:in `run'
/opt/rh/rh-ruby25/root/usr/share/gems/gems/bundler-1.16.1/lib/bundler/cli.rb:424:in `exec'
/opt/rh/rh-ruby25/root/usr/share/gems/gems/bundler-1.16.1/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
/opt/rh/rh-ruby25/root/usr/share/gems/gems/bundler-1.16.1/lib/bundler/vendor/thor/lib/thor/invocation.rb:126:in `invoke_command'
/opt/rh/rh-ruby25/root/usr/share/gems/gems/bundler-1.16.1/lib/bundler/vendor/thor/lib/thor.rb:387:in `dispatch'
/opt/rh/rh-ruby25/root/usr/share/gems/gems/bundler-1.16.1/lib/bundler/cli.rb:27:in `dispatch'
/opt/rh/rh-ruby25/root/usr/share/gems/gems/bundler-1.16.1/lib/bundler/vendor/thor/lib/thor/base.rb:466:in `start'
/opt/rh/rh-ruby25/root/usr/share/gems/gems/bundler-1.16.1/lib/bundler/cli.rb:18:in `start'
/opt/rh/rh-ruby25/root/usr/share/gems/gems/bundler-1.16.1/exe/bundle:30:in `block in <top (required)>'
/opt/rh/rh-ruby25/root/usr/share/gems/gems/bundler-1.16.1/lib/bundler/friendly_errors.rb:122:in `with_friendly_errors'
/opt/rh/rh-ruby25/root/usr/share/gems/gems/bundler-1.16.1/exe/bundle:22:in `<top (required)>'
/opt/rh/rh-ruby25/root/usr/bin/bundle:23:in `load'
/opt/rh/rh-ruby25/root/usr/bin/bundle:23:in `<main>'
(See full trace by running task with --trace)
```

**Comment by ares** [comment]
actually getting the same error when I run `apipie_dsl:static`

**Comment by ares** [comment]
if app/ is needed, it needs to be listed in gemspec at https://github.com/ofedoren/apipie-dsl/blob/master/apipie-dsl.gemspec#L18

**Comment by ares** [comment]
workaround - don't use gem, use git checkout :-)

**Comment by ofedoren** [comment]
Thanks! Fixed in 2.0.0.
